.gitignore

```diff
@@ -127,3 +128,8 @@ dmypy.json
 # pixi environments
 .pixi
 *.egg-info
+
+# lock files
+pixi.lock
+package-lock.json
```

Comment:
I would like to have these in version control at some point. Not having them can lead to a "works on my machine" scenario, which is the whole reason for their existence.